app/javascript/components/search/genes/GeneKeyword.js

```diff
@@ -48,11 +48,13 @@ export default function GeneKeyword({ placeholder, helpTextContent }) {
 
   /** Converts any current typed free text to a gene array entry */
   function syncGeneArrayToInputText() {
-    if (!inputText) {
+    const inputTextTrimmed = inputText.trim().replace(',', '')
```

Comment:
Per Jon, this would handle rare edge cases (typos, really) like "BRCA1,,".

```suggestion
    const inputTextTrimmed = inputText.trim().replace(/,/g, '')
```

Reply:
fixed -- as noted, the edge case isn't possible because the inputText is cleared as soon as a single comma is written, but it can't hurt to be safe!